### PR TITLE
Fix/aws queues

### PR DESCRIPTION
**Pull request by raksiv**
Added queues constructor to the CLI with permissions for compute to work with queues.

### CODE REVIEW

**Review by tjholm — CHANGES_REQUESTED** [review]
Looks mostly good, just a quick comment on the commit history for this branch, as we're using conventional commits in our change logs could you squash/amend to add appropriate fix/feat tags for our changelogs

File: packages/plugins/aws/src/resources/queue.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/**
+ * Nitric AWS Storage Queue based Queue
+ */
+
+export class NitricStorageQueue extends pulumi.ComponentResource {
+	public readonly name: string;
+	public readonly queue: aws.sqs.Queue;
+
+	constructor(name: string, args: NitricStorageQueueArgs, opts?: pulumi.ComponentResourceOptions) {
+		super('nitric:queue:StorageQueue', name, {}, opts);
```
Comment (jyecusch) [inline packages/plugins/aws/src/resources/queue.ts]: Should be `super('nitric:queue:SQSQueue', ...`

File: packages/plugins/aws/src/resources/queue.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		const defaultResourceOptions: pulumi.ResourceOptions = { parent: this };
+		const { queue } = args;
+
+		this.name = queue.name;
+		this.queue = new aws.sqs.Queue(
+			this.name,
+			{
+				contentBasedDeduplication: false,
+				fifoQueue: false,
```
Comment (jyecusch) [inline packages/plugins/aws/src/resources/queue.ts]: What are the default for these values, can we just omit them?

File: packages/plugins/aws/src/resources/compute.ts [hunk 3]
```diff
@@ -104,6 +104,42 @@ export class NitricComputeAWSLambda extends pulumi.ComponentResource {
...
+								'sqs:ListDeadLetterSourceQueues',
+								'sqs:ListQueues',
+								'sqs:ListQueueTags',
+								'sqs:PurgeQueue',
+								'sqs:ReceiveMessage',
+								'sqs:RemovePermission',
+								'sqs:SendMessage',
+								'sqs:SetQueueAttributes',
+								'sqs:TagQueue',
+								'sqs:UntagQueue',
```
Comment (jyecusch) [inline packages/plugins/aws/src/resources/compute.ts]: These permissions probably aren't needed:
- AddPermission
- CreateQueue
- DeleteQueue
- PurgeQueue
- RemovePermission
- SetQueueAttributes
- TagQueue
- UntagQueue

**Review by jyecusch — CHANGES_REQUESTED** [review]

**Review by jyecusch — APPROVED** [review]

**Review by tjholm — APPROVED** [review]

**Pull request merged**